README.md

```diff
@@ -29,14 +35,23 @@ sync:
 
 Required environment:
  * `WATCH_FILE`: file to watch (path relative to volume root)
- * `GIT_COMMIT_MESSAGE`: string or expression evaluated in the volume to provide a commit message 
+ * `GIT_COMMIT_MESSAGE`: string or expression evaluated in the volume to provide a commit message
  * `GIT_USERNAME`: git username for commit
  * `GIT_EMAIL`: git email for commit
 
 To push to a repository, these additional variables are required:
  * `REMOTE_NAME`: name of the git remote, eg `origin`
  * `REMOTE_URL`: git repository URL, eg `https://github.com/fvanderbiest/playground.git`
- * `TOKEN`: password or OAuth token (eg: [GitHub token](https://github.com/settings/tokens))
+
+Optional environment:
+ * `REMOTE_BRANCH`: Remote branch to use
+ * `FORCE_CLONE`: Delete volume content before cloning remote repository
+
+To use SSH authentication to access remote repository, one of following
+variables must be set:
+ * `GIT_RSA_DEPLOY_KEY`: Private RSA key to use (eg: [GitHub deploy keys](https://developer.github.com/guides/managing-deploy-keys/))
+ * `GIT_RSA_DEPLOY_KEY_FILE`: Path to a file containning the private RSA key to use
```

Comment:
containing

Reply:
GIT_RSA_DEPLOY_KEY -> PRIVATE_KEY_FILE
GIT_RSA_DEPLOY_KEY_FILE -> PRIVATE_KEY_STRING ?

Reply:
I would prefer to keep 'RSA' in the var name. What do you think about : 
* GIT_RSA_DEPLOY_KEY -> RSA_PRIVATE_KEY_STRING
* GIT_RSA_DEPLOY_KEY_FILE -> RSA_PRIVATE_KEY_FILE